install/kubernetes/istio-gateway.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
+      number: 15007
+      name: pilot-http-discovery
+      protocol: TCP
+    hosts:
+    - "*"
+  - port:
```

Comment:
For the ingressgateway, we should only expose grpc + tls

If you add another gateway of ILB type - we can expose plain text there.

Reply:
> If you add another gateway of ILB type - we can expose plain text there.

Can you explain what you mean here?

Reply:
Yes, I also want to know why with ILB, we can expose plain text insecure stuff over internet. @costinm

Reply:
Thanks @costinm , got it. Then so you mean that we can specify ILB and LB separately for gateway? I did not found any document for this, can you show some link for ILB and LB configuration for gateway?

/cc @rshriram